templates/prSignUp.html

```diff
@@ -0,0 +1,161 @@
+<!DOCTYPE html>
```

Comment:
You should probably delete this file from your change. 

There's already a migrated [prSignUp.html](https://github.com/mellychat/webapp/blob/mainline/public/templates/prSignUp.html) page in the /public directory. I diffed this version and the one in /public and they don't differ by much (a few new lines). But this version is pointing at the wrong CSS.

Am I missing something?

Reply:
This also seems related to the other PR:

https://github.com/mellychat/webapp/pull/29

I recommend doing all the changes to fix the navbar situation in one PR as these multiple PRs are difficult to track.

Reply:
Hey! Sorry about putting them in multiple PRs. I thought we were supposed to make multiple prs with little changes instead of one pr with large changes. I'll try to rectify that.

This templates folder and the prSignUp should be deleted. It was deleted on my local device, so I don't know how it doesn't show that.

The patientSignup.html and the CSS edits should also remain the same.

Keep the edits in #29 and delete this one. The main difference is the navbar's hrefs have the "/" because the navbar wasn't working without them.

Thanks for your comments!